pkg/controller/managementingress/handler/route.go

```diff
@@ -98,6 +117,16 @@ func (ingressRequest *IngressRequest) CreateOrUpdateRoute() error {
 	}
 	ingressRequest.recorder.Eventf(ingressRequest.managementIngress, "Normal", "CreatedRoute", "Successfully created route %q", RouteName)
 
+	// create ibmcloud-cluster-ca-cert
+	clusterSecret := NewSecret(ClusterSecretName, os.Getenv(PODNAMESPACE), destinationCAcert)
+	err = ingressRequest.Create(clusterSecret)
+	klog.Infof("create secret: %s for %q.", clusterSecret.ObjectMeta.Name, ingressRequest.managementIngress.ObjectMeta.Name)
+	err = ingressRequest.Create(clusterSecret)
```

Comment:
why there is two create operation here? You also need to add owner reference to make sure when user delete the CR the secret can be auto removed by k8s.